README.md

```diff
@@ -1 +1,34 @@
-Refer to respective folders for assignments
\ No newline at end of file
+# Follow the following steps to Execution of Program
+## Prerequisite
+* Need to install python3.X
+* Need to install Mysql and all the plugins needed
+# STEPS
+* 1) Run this script ```pip install -r reqirement.txt```
+* 2) Create The Enviroment Variable using following command
+ * ```setx User_name "Enter Your Database user name"```
+ * ```setx Password "Enter Your Database Password of that user"```
+ * ```setx Port "Enter Your Database Port Number"```
+ * ```setx Db_name "Enter Name of Database of your choice"```
+
+* 3) Run the file named ``` Create_Database.py``` it will create a database if it dose not exist 
+
+* 4) Run the script named ```Create_table_and_intialize.py``` it will create the table if it dose not exist and intialize with values
+
+	* To Schedule the python script to run at every 5 minutes in windows we use Task Scheduler
```

Comment:
Try doing this using [python-crontab](https://pypi.org/project/python-crontab/)?